selfdrive/car/body/carcontroller.py

```diff
@@ -3,29 +3,45 @@
 from common.realtime import DT_CTRL
 from opendbc.can.packer import CANPacker
 from selfdrive.car.body import bodycan
-from selfdrive.car.body.values import SPEED_FROM_RPM
+from selfdrive.car.body.values import CAR, SPEED_FROM_RPM
 from selfdrive.controls.lib.pid import PIDController
 
 
-MAX_TORQUE = 500
+MAX_TORQUE = 700
 MAX_TORQUE_RATE = 50
 MAX_ANGLE_ERROR = np.radians(7)
-MAX_POS_INTEGRATOR = 0.2   # meters
-MAX_TURN_INTEGRATOR = 0.1  # meters
+MAX_POS_INTEGRATOR = 10.0   # meters
+MAX_TURN_INTEGRATOR = 10.0  # meters
+
+MAX_KNEE_TORQUE_LEFT = 400 # knee motor
+MAX_KNEE_TORQUE_RIGHT = 200 # hip motor
```

Comment:
call these knee and hip instead? using left/right seems confusing here. left/right seems like it's just an implementation detail?